scripts/results/plot_fanning.py

```diff
@@ -63,3 +63,23 @@
                         topomap_args=dict(vmin=-300, vmax=300))
 fig.set_size_inches(12, 6, forward=True)
 fig.savefig('FanningB.pdf', bbox_to_inches='tight')
+
+###############################################################################
+# Finally, we can also use the tSSS data which is equivalent to doing a
+# highpass filter and the "fanning" will not be visible also in this case.
```

Comment:
don't say it's "equivalent to", mathematically it's not. You can say "it has a high-passing effect for these data" if you want